PythonAPI/examples/__init__.py

```diff
@@ -0,0 +1,28 @@
+import os, sys, glob
```

Comment:
this is a great idea and why doesn't carla do it already

Reply:
Not sure, I only recently realized we could do this and it makes the PYTHONPATH stuff a lot easier.